core/src/view/view.cpp

```diff
@@ -226,19 +217,19 @@ void View::updateMatrices() {
     // the "top" face of the view frustum with the ground plane
 
     // NOTE: far here can go to infinity and hence a std::min below!
-    float far = m_pos.z / cos(m_pitch + 0.5 * fovy);
+    float far = m_pos.z / std::min(0., cos(m_pitch + 0.5 * fovy));
```

Comment:
The relevant constant here would be `DBL_MIN` (http://www.cplusplus.com/reference/cfloat/), but more importantly I got that comparison totally backwards. Fixing.